starclass/features/freqextr.py

```diff
@@ -401,6 +401,7 @@ def freqextr(lightcurve, n_peaks=6, n_harmonics=0, hifac=1, ofac=4, snrlim=None,
 
 	# Add index to peak number and harmonic for easy lookup:
 	tab.add_index('num')
+	tab.add_index('harmonic')
```

Comment:
Since we are currently not using the indices, we should definitely not add more of them